### ISSUE TITLE
Copyright status

### ISSUE THREAD

**Issue by jonathanrobie** [biblicalhumanities/Nestle1904#6]
The guideline for putting something in the public domain is to use an explicit waiver.  See this:

https://creativecommons.org/choose/zero/waiver

**Comment by jtauber** [comment]
Yes, just saying "public domain" is a problematic notion in some countries including US. CC0 is better (although I still personally prefer CC-BY-SA :-) )

**Comment by jonathanrobie** [comment]
I also prefer CC-BY-SA ... but if the intent is to donate to the public domain, it should be done in a legally sound way.

**Comment by jtauber** [comment]
Yep, and CC0 is the way to do that, not say it's "public domain" (which means something different in different countries)
